java/src/jmri/jmrix/mqtt/MqttTurnout.java

```diff
@@ -12,85 +12,83 @@
 public class MqttTurnout extends AbstractTurnout implements MqttEventListener {
 
     private final MqttAdapter mqttAdapter;
-    private final String topic;
-    private final static String stateTopicPrefix = "state/";
-    private final static String commandTopicPrefix = "command/";
+    private final String sendTopic;
+    private final String rcvTopic;
 
     /**
      * Requires, but does not check, that the system name and topic be consistent
      * @param ma Adapter to reference for connection
      * @param systemName System name of turnout
-     * @param topic MQTT topic being used
+     * @param sendTopicFormat MQTT topic to use when sending (full string, including systemName part)
+     * @param sendTopicFormat MQTT topic to use when receiving (full string, including systemName part)
      */
-    MqttTurnout(MqttAdapter ma, String systemName, String topic) {
+    MqttTurnout(MqttAdapter ma, String systemName, String sendTopic, String rcvTopic) {
         super(systemName);
-        this.topic = topic;
+        this.sendTopic = sendTopic;
+        this.rcvTopic  = rcvTopic;
         mqttAdapter = ma;
-        mqttAdapter.subscribe(getStateTopicPrefix() + topic, this);
-        _validFeedbackNames = new String[] {"DIRECT", "ONESENSOR", "TWOSENSOR", "DELAYED", "MONITORING", "EXACT"};
-        _validFeedbackModes = new int[] {DIRECT, ONESENSOR, TWOSENSOR, DELAYED, MONITORING, EXACT};
+        mqttAdapter.subscribe(rcvTopic, this);  // only receive receive topic, not send one
+        _validFeedbackNames = new String[] {"DIRECT", "ONESENSOR", "TWOSENSOR", "DELAYED", "MONITORING"};
+        _validFeedbackModes = new int[] {DIRECT, ONESENSOR, TWOSENSOR, DELAYED, MONITORING};
         _validFeedbackTypes = DIRECT | ONESENSOR | TWOSENSOR | DELAYED | MONITORING;
     }
 
-    private String getStateTopicPrefix() {
-        return (getFeedbackMode() == EXACT ? stateTopicPrefix : "");
-    }
-
-    private String getCommandTopicPrefix() {
-        return (getFeedbackMode() == EXACT ? commandTopicPrefix : "");
-    }
-
-    @Override
-    public void setFeedbackMode(int mode) throws IllegalArgumentException {
-        mqttAdapter.unsubscribe(getStateTopicPrefix() + topic, this);
-        try {
-            super.setFeedbackMode(mode);
-        } finally {
-            mqttAdapter.subscribe(getStateTopicPrefix() + topic, this);
-        }
-    }
-
     public void setParser(MqttContentParser<Turnout> parser) {
         this.parser = parser;
     }
-    
+        
     MqttContentParser<Turnout> parser = new MqttContentParser<Turnout>() {
         private final static String closedText = "CLOSED";
         private final static String thrownText = "THROWN";
         private final static String unknownText = "UNKNOWN";
         private final static String inconsistentText = "INCONSISTENT";
-        private final static String movingText = "MOVING";
```

Comment:
Noticed now that this state is removed. This is not the same as inconsistent.
MOVING is a controlled state showing that the turnout is switching position. It should not be removed.